master/buildbot/reporters/gerrit.py

```diff
@@ -300,6 +300,9 @@ def buildComplete(self, key, build):
             result = _handleLegacyResult(result)
             self.sendCodeReviews(build, result)
 
+    def checkConfig(self, *args, **kwargs):
```

Comment:
That is already done in [BuildService.__init__](https://github.com/buildbot/buildbot/blob/master/master/buildbot/util/service.py#L174)

I think we should take the previous version of checkConfig.
```python
def checkConfig(self, *args, **kwargs):
    return defer.succeed(True)
```